tools/downloader/src/openmodelzoo/modeltools/converter.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) 2019 Intel Corporation
```

Comment:
```suggestion
# Copyright (c) 2021 Intel Corporation
```

Reply:
It would be incorrect to remove 2019, but I updated the years to include the current year.